packages/lib-classifier/src/plugins/drawingTools/components/Mark/Mark.spec.js

```diff
@@ -197,45 +215,90 @@ describe('Drawing tools > Mark', function () {
   })
 
   describe('useEffect hook', function () {
-    describe('with finished mark and no subTaskVisibility', function () {
-      let markWrapper = (mark) => {
-        return (
+    function markWrapper(mark) {
+      return (
+        <svg>
           <Mark
             label='Point 1'
             mark={mark}
             onDelete={onDelete}
             onFinish={onFinish}
             onSelect={onSelect}
           >
-            <Point mark={point} />
+            <Point mark={mark} />
           </Mark>
-        )
-      }
+        </svg>
+      )
+    }
+
+    describe('with finished mark and no subTaskVisibility', function () {
+      let newMark
+
+      before(function () {
+        window.scrollTo = sinon.stub()
+        newMark = pointTool.createMark()
+        newMark.finish()
+        wrapper = mount(markWrapper(newMark))
+      })
 
-      afterEach(function () {
+      after(function () {
         onFinish.resetHistory()
+        window.scrollTo = undefined
```

Comment:
Was this previously undefined by jsdom? It'd be safer to store the original in a variable and reset this back in case what jsdom does changes in the future. If using sinon's API:
```suggestion
        scrollToStub.restore()
```

Reply:
Yeah, `window.scrollTo` isn't supported by JSDOM. I'm not sure if there's a better workaround for that.